ops/services/web_application_firewall/main.tf

```diff
@@ -13,16 +13,81 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
         variable_name = "RemoteAddr"
       }
       operator           = "GeoMatch"
-      negation_condition = true
-      match_values       = ["US", "CA"]
+      negation_condition = false
+      match_values = [
```

Comment:
Any chance you could link the web pages you used to form this list? I found a partial list.

Reply:
The bulk came from the original ticket: https://github.com/CDCgov/prime-simplereport/issues/3729

Additional entries came from some of my own personal knowledge with the work I do for some entities of DoD and DHS, as well as some conversations with a political scientist. Some of these are not explicitly sanctioned entities, but are included because of their close association to associated entities in one way or another. Unfortunately, there wasn't a single source of truth for this.